Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/LevelRootUiHandler.cpp

```diff
@@ -79,16 +85,76 @@ namespace AzToolsFramework
             return;
         }
 
-        QPen borderLinePen(s_levelRootBorderColor, s_levelRootBorderThickness);
+        // Draw border line at the bottom
+        QPen borderLinePen(m_levelRootBorderColor, PrefabUIConstants::levelDividerThickness);
         QRect rect = option.rect;
-
-        painter->save();
-        painter->setRenderHint(QPainter::Antialiasing, true);
-
-        // Draw border at the bottom
         painter->setPen(borderLinePen);
         painter->drawLine(rect.bottomLeft(), rect.bottomRight());
 
+        if (Prefab::IsPrefabOverridesUxEnabled())
+        {
+            // Draw capsule
+            const bool isFirstColumn = index.column() == EntityOutlinerListModel::ColumnName;
+            const bool isLastColumn = index.column() == EntityOutlinerListModel::ColumnLockToggle;
+
+            QColor backgroundColor = m_prefabCapsuleColor;
+            AZ::EntityId levelContainerEntityId = m_prefabPublicInterface->GetLevelInstanceContainerEntityId();
+
+            if (m_prefabFocusPublicInterface->IsOwningPrefabBeingFocused(levelContainerEntityId))
+            {
+                backgroundColor = m_prefabCapsuleEditColor;
+            }
+            else if (!(option.state & QStyle::State_Enabled))
+            {
+                backgroundColor = m_prefabCapsuleDisabledColor;
+            }
+
+            QPainterPath backgroundPath;
+            backgroundPath.setFillRule(Qt::WindingFill);
+
+            QRect tempRect = option.rect;
+            tempRect.setTop(tempRect.top() + 2);
+            tempRect.setHeight(tempRect.height() - 2); // tweaking capsule height
+
+            if (isFirstColumn)
+            {
+                tempRect.setLeft(tempRect.left() + 1);
+                tempRect.setWidth(tempRect.width() + 1); // remove the gray line between columns
+            }
+
+            if (isLastColumn)
```

Comment:
Can do an "else if" here. Also could move to inside the "if (isFirstColumn || isLastColumn)".